### PR TITLE
fix(inquirer): adding i18n bundle of common module in consumers

### PR DESCRIPTION
**Pull request by cianmSAP**
https://github.com/SAP/open-ux-tools/issues/2496

Exposes function to add the resource bundle for the `@sap-ux/inquirer-common` module
This allows consuming modules to call this after initialising i18next and ensuring all translations are loaded at the time of prompting

### CODE REVIEW

File: packages/ui5-application-inquirer/src/i18n.ts [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
+                defaultProjectNumber
             }
-        },
-        () => i18next.addResourceBundle('en', ui5AppInquirerNamespace, translations)
-    );
+        }
+    });
+    i18next.addResourceBundle('en', ui5AppInquirerNamespace, translations);
+    // add the inquirer common i18n resource bundle to ensure all translations are available
+    addInquirerCommoni18nResourceBundle();
```
Comment (longieirl) [inline packages/ui5-application-inquirer/src/i18n.ts:23]: Do we need to add tests to ensure that this method is executed?

Comment (cianmSAP) [reply packages/ui5-application-inquirer/src/i18n.ts:23]: Tests added

**Review by longieirl — APPROVED** [review]
Thanks @cianmSAP 
Changes look good.
Havent tested locally.
Tests cover change.

**Review by IainSAP — APPROVED** [review]
Thanks @cianmSAP 

- As discussed a follow-up TBI is required to remove the i18n loading weirdness (consuming modules have to load dependant bundles)
- Tests updated as expected
- Did not test locally

**Pull request merged**